test/libsolidity/StandardCompiler.cpp

```diff
@@ -1178,7 +1178,7 @@ BOOST_AUTO_TEST_CASE(evm_version)
 	}
 	// test default
 	result = compile(inputForVersion(""));
-	BOOST_CHECK(result["contracts"]["fileA"]["A"]["metadata"].get<std::string>().find("\"evmVersion\":\"prague\"") != std::string::npos);
+	BOOST_CHECK(result["contracts"]["fileA"]["A"]["metadata"].get<std::string>().find("\"evmVersion\":\"osaka\"") != std::string::npos);
```

Comment:
Let's do this and be done bumping default version in this file forever:

```suggestion
	BOOST_CHECK(result["contracts"]["fileA"]["A"]["metadata"].get<std::string>().find(fmt::format("\"evmVersion\":\"{}\"", EVMVersion::current().name())) != std::string::npos);
```